src/main/webapp/js/feedbackResponseComments.es6

```diff
@@ -70,45 +77,44 @@ var addCommentHandler = function(e) {
                 removeFormErrorMessage(submitButton);
                 addFormRow.prev().show();
                 addFormRow.hide();
-                destroyEditor('responseCommentAddForm-' + responseCommentId);
+                destroyEditor(`responseCommentAddForm-${responseCommentId}`);
             }
-
-        }
+        },
     });
 };
 
-var editCommentHandler = function(e) {
-    var submitButton = $(this);
-    var cancelButton = $(this).next("input[value='Cancel']");
-    var formObject = $(this).closest('form');
-    var displayedText = formObject.siblings("div[id^='plainCommentText']").first();
-    var commentBar = displayedText.parent().find('div[id^=commentBar]');
-    var panelHeading = $(this).parents("[id^='panel_display-']").find('.panel-heading').first();
+const editCommentHandler = (e) => {
+    const submitButton = $(e.currentTarget);
+    const cancelButton = $(e.currentTarget).next("input[value='Cancel']");
+    const formObject = $(e.currentTarget).closest('form');
+    const displayedText = formObject.siblings("div[id^='plainCommentText']").first();
+    const commentBar = displayedText.parent().find('div[id^=commentBar]');
+    const panelHeading = $(e.currentTarget).parents("[id^='panel_display-']").find('.panel-heading').first();
 
     e.preventDefault();
 
-    var commentTextId = formObject.find('div[id^="responsecommenttext-"]').attr('id');
+    const commentTextId = formObject.find('div[id^="responsecommenttext-"]').attr('id');
     formObject.find('input[name=responsecommenttext]').val(tinyMCE.get(commentTextId).getContent());
 
-    var formData = formObject.serialize();
+    const formData = formObject.serialize();
 
     $.ajax({
         type: 'POST',
-        url: submitButton.attr('href') + '?' + formData,
-        beforeSend: function() {
+        url: `${submitButton.attr('href')}?${formData}`,
+        beforeSend() {
             formObject.find('textarea').prop('disabled', true);
             submitButton.html("<img src='/images/ajax-loader.gif'/>");
             submitButton.prop('disabled', true);
             cancelButton.prop('disabled', true);
```

Comment:
Third time in this diff that submit and cancel buttons are disabled, maybe a good candidate for extraction into a small function?